### ISSUE TITLE
Improve/replace the documentation search engine

### ISSUE THREAD

**Issue by engram-design** [doublesecretagency/craft-googlemaps#15]
I managed to figure it out, but the docs search didn't bring anything up for `directions`.

![image](https://user-images.githubusercontent.com/1221575/110588475-2db24700-81c9-11eb-9240-e9a931336e9b.png)

```
{# Old #}
{% set directionsUrl = entry.addressField.directionsUrl(entry.title) %}

{# New #}
{% set directionsUrl = entry.addressField.linkToDirections({ title: entry.title }) %}
```

**Comment by lindseydiloreto** [comment]
Good catch. Looks like Vuepress doesn't index code snippets. I'll add the word "directions" to the description.

Thanks for the heads up! 👍

**Comment by lindseydiloreto** [comment]
So not only does Vuepress not index code snippets, apparently it doesn't index anything that isn't an `<h*>` level header.

<img width="1054" alt="article@2x" src="https://user-images.githubusercontent.com/5309692/112419102-06d73300-8ce8-11eb-9b92-63e2fadd80e6.png">

Which makes more sense, because I was already able to find the word "directions" in an existing paragraph...

<img width="905" alt="directions@2x" src="https://user-images.githubusercontent.com/5309692/112419188-325a1d80-8ce8-11eb-9f6f-a985e9ce27f3.png">

I'm going to do a little more research to figure out if there's a better search mechanism for the Vuepress docs.

Thanks for bringing this to my attention!
